ush/python/pygfs/task/archive.py

```diff
@@ -74,8 +71,9 @@ def configure(self, arch_dict: Dict[str, Any]) -> (Dict[str, Any], List[Dict[str
 
         archive_parm = os.path.join(arch_dict.PARMgfs, "archive")
 
-        # Collect the dataset to archive locally
-        arcdir_j2yaml = os.path.join(archive_parm, "arcdir.yaml.j2")
+        if arch_dict.RUN == "gefs":
+            # Collect the dataset to archive locally
+            arcdir_j2yaml = os.path.join(archive_parm, "gefs_arcdir.yaml.j2")
```

Comment:
As-is, this will break archival of GFS.
```suggestion
        if arch_dict.RUN == "gefs":
            # Collect the dataset to archive locally
            arcdir_j2yaml = os.path.join(archive_parm, "gefs_arcdir.yaml.j2")
        else:
            arcdir_j2yaml = os.path.join(archive_parm, "arcdir.yaml.j2")
```

Really, the existing `arcdir.yaml.j2` should be broken up by run so we can just do `{arch_dict.RUN}_arcdir.yaml.j2`.